fastlane/lib/fastlane/fast_file.rb

```diff
@@ -116,6 +116,11 @@ def platform(platform_name)
       self.current_platform = nil
     end
 
+    # Define environment variables
+    def env(name, value)
+      @runner.add_env(self.current_platform, name, value)
```

Comment:
Should we do some input verifications here? Maybe define `env` with no parameters and print out an error that tells the user how to provide parameters?